corehq/apps/custom_data_fields/dbaccessors.py

```diff
@@ -1,17 +0,0 @@
-from corehq.util.quickcache import quickcache
-
-
-@quickcache(['domain', 'field_type'], timeout=60 * 60)
-def get_by_domain_and_type(domain, field_type):
-    from corehq.apps.custom_data_fields.models import CustomDataFieldsDefinition
-
-    return CustomDataFieldsDefinition.view(
-        'custom_data_fields/by_field_type',
-        key=[domain, field_type],
-        include_docs=True,
-        reduce=False,
-        # if there's more than one,
```

Comment:
Noting there are 25 domain + field type combinations on prod that have multiple docs (including one domain with 4 different LocationFields docs). The extras will need to be manually deleted before the migration is run because of the `unique_together` on the new model.